src/background/responders.ts

```diff
@@ -199,18 +171,42 @@ export const responders: { [T in Action['type']]: Responder<T> } = {
   SCREENSHOT_CAPTURE_FAILURE(): Partial<AppState> {
     return { alert: 'SCREENSHOT FAILURE' }
   },
-  POST_TWEET_SUCCESS(_state, action): Partial<AppState> {
-    const { url } = action.payload.tweetResult
+  POST_TWEET_SUCCESS(state, action): Partial<AppState> {
+    const tab = state.tabs.get(action.payload.tabId)
+    if (!tab) return {}
```

Comment:
How can `tab` not exist? and if it doesn't, shouldn't we still clear the feedback?

Reply:
Maybe you started tweeting and then closed the tab.

Reply:
But we're not checking the tab against the current list of tabs. It's the tab we sent to the action. It'll never be null

Reply:
Nevermind, we're sending the tabId, not the tab. My bad.